schematic/manifest/generator.py

```diff
@@ -1783,6 +1825,44 @@ def get_manifest(
 
         Returns:
             Googlesheet URL, pandas dataframe, or an Excel spreadsheet
+
+        ```mermaid
+        flowchart TD
+            A[Start] --> B{Dataset ID provided?}
```

Comment:
Giving the boxes names instead of `A`, `B`, `C` makes this quite a lot easier to follow and make changes to. That way I can plug it into an LLM and use the full name of the box I might want to change rather than the letter representation of that box.

Reply:
Thanks Bryan. I renamed the letters to box names.